### DISCUSSION TITLE
BeamDyn reference origin

### DISCUSSION THREAD

**Discussion by pelljam** [OpenFAST/openfast#883] [Q&A]
I’m afraid I am far from a proficient OpenFast user, so there is a good chance I’ve failed to understand something here! So, apologies if that is the case. But here is my issue:

I attempted to set up some simple BeamDyn cantilever models. Unfortunately, I don’t have the standalone BeamDyn driver, so I have a fairly convoluted model which hopefully lets the cantilever fall under gravity and then oscillate, but driven through OpenFast.

I have a little spreadsheet that generates mass and stiffness matrices, which I hope conforms to BeamDyn's expectation (but it is possible I have misunderstood that). I have a simple BeamDyn model where I offset the beam’s shear centre -0.7m laterally, the results of which match an equivalent model using another software. I then have a second BeamDyn where I keep the shear centre at 0m and offset the centre of mass and the neutral axis by 0.7m laterally (i.e. to the other side this time). My expectation was that this would not change the physics (although the reporting will change a little) because it is the same system, just with a different reference origin in the cross section. However, when I do this, BeamDyn now reports that the beam's tip will oscillate laterally (B1TipTDxr) with much greater amplitude relative to the original case.

I wondered if you could help me understand this? No doubt I’ve misunderstood something about BeamDyn or its inputs, but it would be nice to know why this "self-consistency" check on the reference origin is not working out.

I have attached the relevant files and a simple spreadsheet I use for generating the input. The model is set to run with BeamDyn_blade.dat, which has the shear centre offset. Also included is BeamDyn_blade_MandNaOffset.dat which has the mass and neutral axis offset. I did this test in v2.4. If you need anything else, please ask.

[Ref study.zip](https://github.com/OpenFAST/openfast/files/6501752/Ref.study.zip)

**Comment by rafmudaf** [comment]
@pelljam the issue you initially created was closed and moved to this discussion.

I'm working on moving all discussion or question and answer issues to Discussions so that we can keep Issues related to things that are actual issues in the software. As I understand your comment, you're asking for more information and an explanation of the BeamDyb reference frame. If you've intended to report an issue, I'm happy to reopen the original issue.

**Reply by pelljam** [reply]
@rafmudaf, ah OK I understand. Tbh I was not sure if it was an issue or not; my observation was that the physical response was dependent on the choice of reference axis origin, which I assumed would be arbitrary - so, in that sense it is an issue. However, it could be I am simply wrong or that that assumption is wrong.

Anyway, I think issue #818 is probably reporting the same thing anyway. So maybe you don't need two issues.
